persistence/test/setup_test.go

```diff
@@ -161,9 +161,9 @@ func fuzzSingleProtocolActor(
 		switch op {
 		case "UpdateActor":
 			numParamUpdatesSupported := 3
-			newStakedTokens := originalActor.StakedTokens
+			newStakedTokens := originalActor.StakedAmount
 			newChains := originalActor.Chains
-			newActorSpecificParam := originalActor.ActorSpecificParam
+			newActorSpecificParam := originalActor.GenericParam
```

Comment:
Not a blocker to merge this in, but just starting a thread for a quick convo

We use `ActorSpecificParam` or `GenericParam` to represent `MaxRelays`, `ServiceURI`, etc... We had https://github.com/pokt-network/pocket/issues/206 originally to do some consolidation, but are you okay with us openining another ticket to improve this in the future? 

Doesn't even have to be an M1 tasks, but maybe more M3 or M4 related once things are more mature.

Reply:
Yes - I noticed this as well. I think we need another consolidation / improve ticket. 

Are you thinking strictly consolidation, if so I can open the tix - else I might need your help to craft it

Reply:
Crafted here: #313